core/controllers/practice_sessions_test.py

```
@@ -505,6 +505,289 @@ def test_get_succeeds_with_arc_id(self) -> None:
+                        'title': 'Default Arc',
+                        'description': 'Default arc',
+                        'node_ids': ['node_1'],
+                    }
+                ),
+            ],
+            'Added acquired skill IDs and arc.',
+        )
+
+        json_response = self.get_json(
+            '%s/staging/%s/arc/default_arc'
+            % (feconf.PRACTICE_SESSION_DATA_URL_PREFIX, 'public-topic-name'),
```

Ruff (0.16.1)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)